docs/overview/usage.md

```diff
@@ -0,0 +1,34 @@
+## Check your usage
+
+To check your usage, sign in to your developer account.
+
+1. Sign in at https://mapzen.com/developers.
+2. Click the statistics button for the API key you want to review.
+3. View a graph of your recent usage for a certain period of time, such as the past day or month.
+
+You also receive [HTTP status codes](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes) in the header for the server's response to your query.
+
+If you exceed rate limits, you will typically see errors for 403 Forbidden and 429 Too Many Requests.
+
+Mapzen uses server caching to deliver commonly requested content as quickly as possible. Queries that are served from a cache do not count toward your rate limits. For example, you might encounter results from the cache when you browse a map with vector tiles in a popular extent or repeatedly perform an identical geocoding search.
```

Comment:
We should ditch this paragraph entirely — it’s TMI for users, may be confusing, and shouldn’t be documented behavior.